docs/authoring.md

```diff
@@ -86,78 +86,132 @@ To make your component available as a NuGet package, it is important to include
+            <ActivatableClass ActivatableClassId="MyAuthoredComponent.Class1" ThreadingModel="both" />
+            <ActivatableClass ActivatableClassId="MyAuthoredComponent.Class2" ThreadingModel="both" />
+          </InProcessServer>
+         </Extension>
+      </Extensions>
+      ```
 
-2. Add your activatable class registrations to the manifest file. Below is an example with a C#/WinRT component `MyAuthoredComponent` and activatable classes `WinRTComponent.Class*`. Note that `WinRT.Host.dll` must be specified as the file name.
+      See the [Simple C#/WinRT component sample](../src/Samples/AuthoringDemo) (specifically the WinUI3CppApp) for an example.
 
-      ```xml
-      <?xml version="1.0" encoding="utf-8"?>
-      <assembly manifestVersion="1.0" xmlns="urn:schemas-microsoft-com:asm.v1">
-      <assemblyIdentity version="1.0.0.0" name="CppConsoleApp"/>
-      <file name="WinRT.Host.dll">
-      <activatableClass
-            name="MyAuthoredComponent.Class1"
-            threadingModel="both"
-            xmlns="urn:schemas-microsoft-com:winrt.v1" />
-      <activatableClass
-            name="MyAuthoredComponent.Class2"
-            threadingModel="both"
-            xmlns="urn:schemas-microsoft-com:winrt.v1" />
-      </file>
-      </assembly>
-      ```
+  - For **unpackaged C++ apps**: The app consumer needs to create an [application manifest file](https://docs.microsoft.com/windows/win32/sbscs/application-manifests) (also known as a "fusion manifest") that follows the naming: `<MyNativeAppName>.exe.manifest`. See this [sample manifest file](https://github.com/microsoft/CsWinRT/tree/master/src/Samples/AuthoringDemo/CppConsoleApp/CppConsoleApp.exe.manifest) for an example.
 
-3. Modify the project to include the manifest file in the output when deploying the project. Right-click on the file in **Solution Explorer**, select **Properties**, and set the **Content** property to **True** using the drop-down arrow on the right.
+      Follow these steps to add an application manifest file to your project:
 
-### Consuming from C# applications
+      1. In Visual Studio, right click on the project node under **Solution Explorer** and click **Add -> New Item**. Search for the **Text File** template and name your file `<MyNativeAppName>.exe.manifest`.
 
-Consuming a C#/WinRT component from C#/.NET apps is supported by both package reference or project reference. This scenario is equivalent to consuming any ordinary C# class library and does not involve WinRT activation in most cases.
+      2. Add your activatable class registrations to the manifest file. Below is an example with a C#/WinRT component called `MyAuthoredComponent` and activatable classes `WinRTComponent.Class*`. Note that `WinRT.Host.dll` must be specified as the file name, without a relative path.
 
-**Note:** With C#/WinRT 1.3.5 or later, project references to a C#/WinRT component require .NET 6.
+            ```xml
+            <?xml version="1.0" encoding="utf-8"?>

```

Comment:
does the manifest not need a `<supportedOS>` entry for unpackaged?

Reply:
I'm not aware of this, perhaps thats why the unpackged scenario doesn't work? When I made the unpackaged scenario I followed the [docs here](https://docs.microsoft.com/en-us/windows/apps/winui/winui3/create-your-first-winui3-app?pivots=winui3-unpackaged-cpp#unpackaged-create-a-new-project-for-an-unpackaged-c-or-c-winui-3-desktop-app).

Reply:
also @stevewhims for context